README.md

```diff
@@ -3,15 +3,3 @@
 This is an algorithm for the [House Prices - Advanced Regression Techniques competition on Kaggle](https://www.kaggle.com/c/house-prices-advanced-regression-techniques).
 I used Kaggle's "Intro to Machine Learning" course as a core supplemented by various online resources to teach myself
 the basics of machine learning and data analysis as well as apply them in an easy to see and evaluate manner.
-
-## User Comments:
```

Comment:
Why did you remove this? This seems like a good feature for our customers

Reply:
Please view the lines below.